pom.xml

```diff
@@ -31,10 +31,10 @@
         <nexus.repository.base>/nexus</nexus.repository.base>
 
         <!-- Please keep (version) properties in alphabetical order! -->
-        <jboss.logging.version>3.3.2.Final</jboss.logging.version>
+        <jboss.logging.version>3.4.1.Final</jboss.logging.version>
         <junit.version>4.12</junit.version>
-        <keycloak.version>4.8.3.Final</keycloak.version>
-        <resteasy.version>3.6.2.Final</resteasy.version>
+        <keycloak.version>8.0.1</keycloak.version>
+        <resteasy.version>3.9.1.Final</resteasy.version>
```

Comment:
Think about adding a `dependencyManagement` section, referencing the `keycloak-bom`. Then you don't have to think about dependent versions, just use the dependencies.

Like this:
```
<dependencyManagement>
  <dependencies>
    <dependency>
      <groupId>org.keycloak.bom</groupId>
      <artifactId>keycloak-adapter-bom</artifactId>
      <version>${keycloak.version}</version>
    </dependency>
  </dependencies>
</dependencyManagement>
```

Reply:
Sounds good. Please add it directly to master if you don't mind - no need to work extensively with PRs ...

Reply:
done.